### PR TITLE
scriptworker 38

### PR DESCRIPTION
**Pull request by escapewindow**
This should partially unblock upgrading the firefoxci cluster to tc 41.x.
I also need to roll out scriptworker 38.0.0 to the mac signers.

### CODE REVIEW

File: addonscript/requirements/base.py37.txt [hunk 1]
```diff
@@ -51,9 +51,9 @@ aiomemoizettl==0.0.3 \
     --hash=sha256:07a6becac60f6cd2604b9f2b73bcd9a50079a0b7b55e2a4e45b1eec5a3ea9659 \
     --hash=sha256:0a80d2dc765e545263f515363b6700ec8cf86fa3968b529f56390b28e34f743d
     # via scriptworker
-arrow==1.0.3 \
-    --hash=sha256:3515630f11a15c61dcb4cdd245883270dd334c83f3e639824e65a4b79cc48543 \
-    --hash=sha256:399c9c8ae732270e1aa58ead835a79a40d7be8aa109c579898eb41029b5a231d
+arrow==1.1.0 \
```
Comment (escapewindow) [inline addonscript/requirements/base.py37.txt:54]: https://arrow.readthedocs.io/en/latest/releases.html#id1 looks ok.

File: addonscript/requirements/base.py37.txt [hunk 2]
```diff
@@ -62,9 +62,9 @@ async-timeout==3.0.1 \
     #   -r requirements/base.in
     #   aiohttp
     #   taskcluster
-attrs==20.3.0 \
-    --hash=sha256:31b2eced602aa8423c2aea9c76a724617ed67cf9513173fd3a4f03e3a929c7e6 \
-    --hash=sha256:832aa3cde19744e49938b91fea06d69ecb9e649c93ba974535d08ad92164f700
+attrs==21.2.0 \
```
Comment (escapewindow) [inline addonscript/requirements/base.py37.txt:65]: Quite a lot's going on [here](https://www.attrs.org/en/stable/changelog.html#id1) but it looks like the only backwards-incompatible stuff is for py34 and between 21.1.0 -> 21.2.0 ?

File: addonscript/requirements/base.py37.txt [hunk 3]
```diff
@@ -279,13 +279,13 @@ rsa==4.7.2 \
     --hash=sha256:78f9a9bf4e7be0c5ded4583326e7461e3a3c5aae24073648b4bdfa797d78c9d2 \
     --hash=sha256:9d689e6ca1b3038bc82bf8d23e944b6b6037bc02301a574935b2dd946e0353b9
     # via python-jose
-scriptworker==37.0.3 \
-    --hash=sha256:7739a8cae17064d4f5763a5bc390e99637b7e5d6cec6a46c6ed403935cdced0a \
-    --hash=sha256:a19c1b17d60ce39954a914e35e5a3a53fa0d900d73cd512160a3fb2299ce69c5
+scriptworker==38.0.0 \
```
Comment (escapewindow) [inline addonscript/requirements/base.py37.txt:282]: Drops py36 support

File: addonscript/requirements/test.py37.txt [hunk 4]
```diff
@@ -25,14 +26,14 @@ check-manifest==0.46 \
     --hash=sha256:5895e42a012989bdc51854a02c82c8d6898112a4ab11f2d7878200520b49d428 \
     --hash=sha256:b59b0e7c7ed3946537677c9ab9b2c2cb7be9b1807fd40bc4dfc1eef31d42cff5
     # via -r requirements/test.in
-click==7.1.2 \
-    --hash=sha256:d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a \
-    --hash=sha256:dacca89f4bfadd5de3d7489b7c8a566eee0d3676333fbb50030263894c38c0dc
+click==8.0.0 \
```
Comment (escapewindow) [inline addonscript/requirements/test.py37.txt:29]: [changelog](https://click.palletsprojects.com/en/8.0.x/changes/#version-8-0-0)

File: beetmoverscript/requirements/base.py37.txt [hunk 5]
```diff
@@ -190,59 +190,41 @@ jwcrypto==0.8 \
...
-    --hash=sha256:caabedc8323f1e93231b52fc32bdcde6db817623d33e100708d9a68e1f53b26b \
-    --hash=sha256:cd5df75523866410809ca100dc9681e301e3c27567cf498077e8551b6d20e42f \
-    --hash=sha256:cdb132fc825c38e1aeec2c8aa9338310d29d337bebbd7baa06889d09a60a1fa2 \
-    --hash=sha256:d53bc011414228441014aa71dbec320c66468c1030aae3a6e29778a3382d96e5 \
-    --hash=sha256:d73a845f227b0bfe8a7455ee623525ee656a9e2e749e4742706d80a6065d5e2c \
-    --hash=sha256:d9be0ba6c527163cbed5e0857c451fcd092ce83947944d6c14bc95441203f032 \
-    --hash=sha256:e249096428b3ae81b08327a63a485ad0878de3fb939049038579ac0ef61e17e7 \
-    --hash=sha256:e8313f01ba26fbbe36c7be1966a7b7424942f670f38e666995b88d012765b9be \
-    --hash=sha256:feb7b34d6325451ef96bc0e36e1a6c0c1c64bc1fbec4b854f4529e51887b1621
+markupsafe==2.0.0 \
```
Comment (escapewindow) [inline beetmoverscript/requirements/base.py37.txt:193]: [changelog](https://markupsafe.palletsprojects.com/en/2.0.x/changes/#version-2-0-0)

File: beetmoverscript/requirements/base.py37.txt [hunk 6]
```diff
@@ -347,17 +329,17 @@ requests==2.25.1 \
     # via
     #   github3.py
     #   taskcluster
-s3transfer==0.3.7 \
-    --hash=sha256:35627b86af8ff97e7ac27975fe0a98a312814b46c6333d8a6b889627bcd80994 \
-    --hash=sha256:efa5bd92a897b6a8d5c1383828dca3d52d0790e0756d49740563a3fb6ed03246
+s3transfer==0.4.2 \
```
Comment (escapewindow) [inline beetmoverscript/requirements/base.py37.txt:332]: [changelog](https://github.com/boto/s3transfer/blob/develop/CHANGELOG.rst#042)

File: beetmoverscript/requirements/base.txt [hunk 7]
```diff
@@ -165,9 +165,9 @@ immutabledict==1.3.0 \
     --hash=sha256:b52f880323b3e94e763d5a09930587cce68387e497a4ec669ae836c1bd8cdd1e \
     --hash=sha256:e8dc4d8ba91de43d703c123b9e99f7029901265d3ee0f0a01cf17e3e6981bbf9
     # via scriptworker
-jinja2==2.11.3 \
-    --hash=sha256:03e47ad063331dd6a3f04a43eddca8a966a26ba0c5b7207a9a9e4e08f1b29419 \
-    --hash=sha256:a6d58433de0ae800347cab1fa3043cebbabe8baa9d29e668f1c768cb87a333c6
+jinja2==3.0.0 \
```
Comment (escapewindow) [inline beetmoverscript/requirements/base.txt:168]: [changelog](https://jinja.palletsprojects.com/en/3.0.x/changes/#version-3-0-0)

**Review by escapewindow — COMMENTED** [review]
Made it through bouncerscript

**Review by bhearsum — APPROVED** [review]

**Pull request merged**